docs/source/conf.py

```diff
@@ -52,14 +54,37 @@
 copyright = u'2016, Palantir Technologies'
 author = u'Palantir Technologies'
 
+def get_version_from_gradle():
+  """ The current version of this codebase.
+
+  Parses the output of ./gradlew printVersion in order to get the current
+  version of the repository. We are assuming that this file is two directories
+  up from the codebase root, and the version is the line after ':printVersion'
+  in the stdout of the gradle command.
+
+  Returns:
+    A string representing full version.
+
+  Raises:
+    A variety of exceptions if either the parsing or command fails.
+  """
+  import subprocess, os
+  conf_py_directory = os.path.dirname(os.path.realpath(__file__))
+  codebase_root = os.path.join(conf_py_directory, '..', '..')
+  p = subprocess.Popen(['./gradlew', 'printVersion'],
+                       cwd=codebase_root, stdout=subprocess.PIPE)
+  stdout = p.communicate()[0]
+  lines = stdout.split('\n')
+  return lines[lines.index(':printVersion') + 1]
+
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = 'develop'
+version = get_version_from_gradle()
```

Comment:
This string is also used in the menu bar of the docs. We should think about what we want to display there.

Also, it looks like `|release|` and `|version|` are identical - does it come out as `0.25.0` or something like `0.25.0+9gaf3452d`?